src/workloads/execution/TimeSeriesSort.yml

```diff
@@ -0,0 +1,594 @@
+          }},
+          {$_internalBoundedSort: {
+            sortKey: {t: -1},
+            bound: {base: "max"},
+          }},
+          {$_internalInhibitOptimization: {}},
+          {$skip: 1e10}
+        ]
+        cursor: {batchSize: *batchSize}
+  - *Nop
+  - *Nop
+  - Repeat: *numQueryRuns
+    Database: *db
+    Operations:
+    - OperationMetricsName: TimeToFirstResultQueryBlockingSort
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: *coll
+        pipeline: [{$sort: {t: 1}}, {$_internalInhibitOptimization: {}}, {$limit: 1}]
+        cursor: {batchSize: *batchSize}
+  - *Nop
+  - Repeat: *numQueryRuns
+    Database: *db
+    Operations:
+    - OperationMetricsName: TimeToFirstResultQueryBoundedSorter
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: system.buckets.Collection0
+        pipeline: [
+          {$_internalUnpackBucket: {
+            exclude: [],
+            timeField: "t",
+            metaField: "m",
+            bucketMaxSpanSeconds: 3600,
+            assumeNoMixedSchemaData: true
+          }},
+          {$_internalBoundedSort: {
+            sortKey: {t: 1},
+            bound: {base: "min"},
+          }},
+          {$limit: 1}
+        ]
+        cursor: {batchSize: *batchSize}
+        hint: {$natural: 1}
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: *numQueryRuns
+    Database: *db
+    Operations:
+    - OperationMetricsName: MetaPointPredicateTimeSortBlockingSort
+      OperationName: RunCommand
+      OperationCommand:
+        explain:
+          aggregate: *coll
+          pipeline: [
+            {$match: {m: 1001}},  # this will match ~5000 docs
+            {$sort: {t: 1}},
+            {$_internalInhibitOptimization: {}},
+            {$skip: 1e10}
+          ]
+          cursor: {batchSize: *batchSize}
+  - *Nop
+  - Repeat: *numQueryRuns
+    Database: *db
+    Operations:
+    - OperationMetricsName: MetaPointPredicateTimeSortBoundedSorter
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: system.buckets.Collection0
+        pipeline: [
+          {$match: {meta: 1001}},
+          {$sort: {"control.min.t": 1}},
+          {$_internalUnpackBucket: {
+            exclude: [],
+            timeField: "t",
+            metaField: "m",
+            bucketMaxSpanSeconds: 3600,
+            assumeNoMixedSchemaData: true
+          }},
+          {$_internalBoundedSort: {
+            sortKey: {t: 1},
+            bound: {base: "min"},
+          }},
+          {$_internalInhibitOptimization: {}},
+          {$skip: 1e10}
+        ]
+        cursor: {batchSize: *batchSize}
+  - *Nop
+  - *Nop
+  - Repeat: *numQueryRuns
+    Database: *db
+    Operations:
+    - OperationMetricsName: SelectiveMetaPointPredicateTimeSortBlockingSort
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: *coll
+        pipeline: [
+          {$match: {m: 1002}},  # this will match ~10 docs
```

Comment:
This will match exactly 10 docs, right?  Because the only insert phase where m is 1002 generates 10 docs.

Reply:
Oh yea I can change that